sqlx-sqlite/README.md

```diff
@@ -0,0 +1,58 @@
+# DataFusion secondary index example using SQLx
+
+This example demonstrates how to integrate a secondary index built using SQLite via SQLx with DataFusion.
+
+SQLite is used as a stand-in for an external remote relational database, it should be easy to adapt this example to use another database.
+
+This examples should be considered incomplete: it does not try to handle **many** edge cases or push down filters as much as possible.
```

Comment:
In particular, I am quite concerned that the filter pushdown might be incorrect in some ways, I don't intend for people to copy this or use it directly. Happy to add some tests and increase confidence then tone down this warning.

Reply:
I'll point out any obvious errors I see